test/registry.js

```diff
@@ -84,4 +84,9 @@ contract('Registry', (accounts) => {
     assert.equal(web3.toAscii(register.logs[1].args.contentType).replace(/\u0000/g, ''), 'MusicGroup');
     assert.equal(register.logs[1].args.cid, 'Qmb');
   });
+  it('record previous block number', async () => {
+    var number = web3.eth.blockNumber;
+    const register = await registry.publish([accounts[1], accounts[1]], ['delete', 'create'], ['MusicGroup', 'MusicGroup'], 'Qma|Qmb', { from: accounts[1] });
+    assert.equal(register.logs[0].args.prevblock, number);
+  })
```

Comment:
Can you also add a test that checks multiple registrations in one transaction and see if it also works correctly? Additionally, can you please lint as well (Solidity & JS linting). :)